### PR TITLE
[Added] Blanket Order-Added description for order type 'purchasing'

### PR DESCRIPTION
**Pull request by ShraddhaRanjane**

### CODE REVIEW

**Comment by ShraddhaRanjane** [comment]
I have added a few descriptions in  Blanket Order for order type-Purchasing.

File: erpnext_com/www/docs/user/manual/en/selling/blanket-order.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ To access Blanket Order, go to:
 ## 1. Prerequisites
 Before creating and using a Blanket Order, it is advised that you create the following first:
 
-* [Customer](/docs/user/manual/en/CRM/customer)
+* [Customer](/docs/user/manual/en/CRM/customer)/[Supplier](/docs/user/manual/en/buying/supplier)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/selling/blanket-order.md]:
```suggestion
* [Customer](/docs/user/manual/en/CRM/customer) or [Supplier](/docs/user/manual/en/buying/supplier)
```

File: erpnext_com/www/docs/user/manual/en/selling/blanket-order.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ To access Blanket Order, go to:
...
 * [Item](/docs/user/manual/en/stock/item)
 
-## 2. How to Create a Blanket Order
+## 2. How to Create a Blanket Order for Sales as well as Purchase
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/selling/blanket-order.md]:
```suggestion
## 2. How to Create a Blanket Order for Sales and Purchase
```

File: erpnext_com/www/docs/user/manual/en/selling/blanket-order.md [hunk 3]
```diff
@@ -33,9 +33,20 @@ The Ordered Quantity field will be updated once a Sales Order is placed. Once a
 
 ### 3.2 View Sales Orders Created against Blanket Order
 
-Click on View Orders button to view the list of sales orders created against this blanket order.  
+Click on View Orders button to view the list of sales orders created against this blanket order. 
+
+### 3.3 Create Purchase Orders
+
+The Ordered Quantity field will be updated once a Purchase Order is placed. Once a blanket order is submitted, you can create new purchase orders by clicking on the Create Purchase Order button.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/selling/blanket-order.md]:
```suggestion
The 'Ordered Quantity' field will be updated once a Purchase Order is placed. Once a Blanket Order is submitted, you can create new Purchase Orders by clicking on the **Create Purchase Order** button.
```

File: erpnext_com/www/docs/user/manual/en/selling/blanket-order.md [hunk 4]
```diff
@@ -33,9 +33,20 @@ The Ordered Quantity field will be updated once a Sales Order is placed. Once a
...
+
+### 3.4 View Purchase Orders Created against Blanket Order
+
+Click on View Orders button to view the list of purchase orders created against this blanket order.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/selling/blanket-order.md]:
```suggestion
Click on the **View Orders** button to view the list of Purchase Orders created against this Blanket Order.
```

File: erpnext_com/www/docs/user/manual/en/selling/blanket-order.md [hunk 5]
```diff
@@ -33,9 +33,20 @@ The Ordered Quantity field will be updated once a Sales Order is placed. Once a
...
+
+**Note-** Multiple Sales Order and Purchase Order can be created against one Blanket Order.
+
 
 ### 4. Related Topics
-* [Sales Order](/docs/user/manual/en/selling/sales-order)
+* [Sales Order](/docs/user/manual/en/selling/sales-order)/[Purchase Order](/docs/user/manual/en/buying/purchase-order)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/selling/blanket-order.md]:
```suggestion
* [Sales Order](/docs/user/manual/en/selling/sales-order)
* [Purchase Order](/docs/user/manual/en/buying/purchase-order)
```

**Comment by ShraddhaRanjane** [comment]
@prasadarr Suggested changes has been done.please check.

**Pull request merged**